test/test_parse_az.py

```diff
@@ -0,0 +1,429 @@
+#
+# Copyright 2017 Mycroft AI Inc.
```

Comment:
Would be good to use the current year in each of these license headers.

Reply:
Ok, I will fix them.